### DISCUSSION TITLE
New Feature Request: Pivot Grid

### DISCUSSION THREAD

**Discussion by itsmehedihasan** [primefaces/community#474] [PrimeNG] [answered]
Hi, 
Is there any plan / possibility to implement Pivot Grid like [ AG > Pivot grid](https://www.ag-grid.com/javascript-data-grid/pivoting/) or [DevExtreme PivotGrid](https://js.devexpress.com/Angular/Demos/WidgetsGallery/Demo/PivotGrid/) ? 
It would be a great feature for complex data visibility or generating report. 

Thanks & appreciate your hard work !!

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We have plans to make a Pivot Table in the future.

**Comment by parked-toes** [comment]
I'm evaluating which component to use for pivoting too.  What about to extend TreeTable component? Mostly I'm missing column expand/collapse.
